packages/@react-aria/grid/src/useGrid.ts

```diff
@@ -98,6 +98,35 @@ export function useGrid<T>(props: GridProps, state: GridState<T, GridCollection<
     scrollRef
   });
 
+  const cachedFocusedKey = useRef(null);
+  const cachedCollection = useRef(null);
+  useEffect(() => {
+    if (
+      state.collection.size > 0 &&
+      state.selectionManager.focusedKey === null &&
+      cachedFocusedKey.current !== null
+    ) {
+      const node = cachedCollection.current.getItem(cachedFocusedKey.current);
+      const isCell = node.type === 'cell' || node.type === 'rowheader' || node.type === 'column';
+      const parentNode =
+        node.parentKey && isCell ?
```

Comment:
Keys can be falsy (e.g. `0`)

```suggestion
        node.parentKey != null && isCell ?
```